bin/wasm-node/rust/src/ffi/bindings.rs

```diff
@@ -243,8 +243,8 @@ pub extern "C" fn init(
 ///
 /// Responses and subscriptions notifications are sent back using [`json_rpc_respond`].
 #[no_mangle]
-pub extern "C" fn json_rpc_send(text_ptr: u32, text_len: u32) {
-    super::json_rpc_send(text_ptr, text_len)
+pub extern "C" fn json_rpc_send(text_ptr: u32, text_len: u32, chain_index: u32) {
+    super::json_rpc_send(text_ptr, text_len, chain_index)
 }
 
```

Comment:
You also need to modify the `json_rpc_respond` function to pass back the chain index as well.